platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/connectivity/SocketFactory.java

```diff
@@ -126,4 +128,12 @@ static void configureAndConnect(
      */
     @NonNull
     Socket createClientSocket(@NonNull final String hostname, final int port) throws IOException;
+
+    /**
+     * A convenient handler for reloading connectivity components
+     * e.g. it could be used to reload a TLS trust store, or ask peers to re-authenticate
```

Comment:
```suggestion
    /**
     * Update the current list of peers with which we wish to be connect.
```

Reply:
on a second thought, this method isn't updating the list of peers. It's reloading the trust store with certificates presented by the peers. Maybe 
`   * reloads the trust store with peer certificates` 
is better?